activerecord/lib/active_record/enum.rb

```diff
@@ -119,8 +132,9 @@ def serialize(value)
       attr_reader :name, :mapping
     end
 
-    def enum(definitions)
+    def enum(definitions, options = {})
```

Comment:
@kaspth can you please explain how you see api with keyword arguments? cause if we don't want to brake api we can't avoid brackets because any function with more than one argument and first argument is hash need them. Sure we can do `def enum(prefix: nil, **definitions)` but with that we reserve prefix and user can not use it as `enum`.